climada/hazard/trop_cyclone/trop_cyclone_windfields.py

```diff
@@ -117,16 +118,25 @@ def compute_angular_windspeeds(
         Wind profile model selection according to MODEL_VANG.
     model_kwargs: dict, optional
         If given, forward these kwargs to the selected model. Default: None
-    cyclostrophic : bool, optional
-        If True, do not apply the influence of the Coriolis force (set the Coriolis terms to 0).
-        Default: False
+    cyclostrophic: bool, optional, deprecated
+        This argument is deprecated and will be removed in a future release.
+        Include `cyclostrophic` as `model_kwargs` instead.
 
     Returns
     -------
     ndarray of shape (npositions, ncentroids)
         containing the magnitude of the angular windspeed per track position per centroid location
     """
     model_kwargs = {} if model_kwargs is None else model_kwargs
+
+    if cyclostrophic is not None:
+        warnings.warn(
+            "The 'cyclostrophic' argument is deprecated and will be removed in a future"
+            "release. Include it in 'model_kwargs' instead.",
+            DeprecationWarning,
+        )
+        model_kwargs["cyclostrophic"] = cyclostrophic
+
```

Comment:
> The proper way to deal with this is probably to mark the argument as deprecated rather than removing it right away. So we can keep it compatible. If it is provided despite its deprecation insert it in `model_kwargs`, e.g., like this:
> 
> ```python
> if cyclostrophic is not None:
>     warnings.warn("yada yada", DeprecationWarning)
>     model_kwargs["cyclostrophic"] = cyclostrophic
> ```

Like this @emanuel-schmid ? see also the other edits